tests/framework/Optimizers/BayesianOptimizer/Basic/BayesianBasic.xml

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/Optimizers.BayesianBasic</name>
+    <author>grifanth</author>
+    <created>2023-06-20</created>
+    <classesTested>BayesianOptimizer</classesTested>
+    <description>
+      This is the most basic test of the Optimizer. It optimizes a basic test function with the Bayesian Optimizer.
+      This test provides a baseline for functionality and regression testing of the method. Particular options tested include
+      custom and anisotropic kernels alongside the LCB acquisition function solved via differential evolution.
+    </description>
+    <analytic>
+      This test uses a basic tests function, which is nonconvex and continuous for testing the
+      optimizer's ability to find global solutions.
+    </analytic>
```

Comment:
Is it test an analytic test? If so, it is better to document it. Such as convergence behavior, optimal point, etc.